packages/foodie-backend/src/app/auth/AuthActions.ts

```diff
@@ -1,7 +1,8 @@
 import { validate } from 'joi';
```

Comment:
Note that joi has been moved to @hapi/joi

Reply:
yeah I just reaizefd that day before yesterday, would move all references to plain joi out